src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$clock-size: 660px;
```

Comment:
Consider using a more descriptive variable name for the clock size, such as $stopwatch-size, to better reflect the task requirement.